File: C++/1378.cpp
#include <iostream>

using namespace std;

int main(){
	int n, sum = 0, res = 0;
	cin >> n;
	
	for(int i = 1; i <= n; i++){
		sum = 0;
		for(int j = 1; j <= i; j++){
			sum += j;
		}
		res += sum;
	}
	cout << res;
}

File: Java/1378.java
import java.util.Scanner;

public class Main{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int n = sc.nextInt();
		int sum = 0;
		int res = 0;
		
		for(int i = 1; i <= n; i++){
			sum = 0;
			for(int j = 1; j <= i; j++){
				sum += j;
			}
			res += sum;
		}
		
		System.out.println(res);
	}
}